<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManageAvailabilityPage</class>
 <widget class="QWidget" name="ManageAvailabilityPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>417</width>
    <height>324</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>391</width>
     <height>201</height>
    </rect>
   </property>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <column>
    <property name="text">
     <string>Mon</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Tue</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Wed</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Thu</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Fri</string>
    </property>
   </column>
   <item row="0" column="0">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="1" column="0">
    <property name="text">
     <string/>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEdit_worker_id">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>201</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Worker ID</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_confirm">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>83</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_return">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>361</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_search">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>search</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
